hack/deployment-diagnostics.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ -z "${EV2:-}" ]]; then
+   # this script only executes in EV2
+   # executing this in other environments with less restricted access to CD logs
+   # can lead to leaking sensitive information
```

Comment:
n.b. anyone who can send a PR that edits files under `.github` has an attack vector for this

Reply:
i checked that only project member PRs actually run PR checks